pkg/registry/core/pod/strategy.go

```diff
@@ -292,39 +295,38 @@ var ResizeStrategy = podResizeStrategy{
 
 // dropNonResizeUpdates discards all changes except for pod.Spec.Containers[*].Resources, pod.Spec.InitContainers[*].Resources, ResizePolicy and certain metadata
 func dropNonResizeUpdates(newPod, oldPod *api.Pod) *api.Pod {
```

Comment:
should the resetFieldsFilter matcher also include initContainers?

Reply:
yes, I think so. Updated.